obspy/io/sac/core.py

```diff
@@ -381,7 +381,8 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
     """
     # read SAC file
-    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize)
+    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize,
+                        encoding=kwargs.get('encoding', 'ASCII'))
```

Comment:
@jkmacc-LANL  Which means if I change `_internal_read_sac` 
From 
```Python
def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
                       **kwargs):  # @UnusedVariable
```
To
```Python
def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
                       encoding='ASCII', **kwargs):  # @UnusedVariable
```
I don't need to worry about breaking those two functions in the `obspy.core` module?